docs_2.0/manual_EN/4.ngql-guide/16.query-tuning-statements/1.explain-and-profile.md

```diff
@@ -0,0 +1,112 @@
+# EXPLAIN and PROFILE
+
+`EXPLAIN` helps output the execution plan of an nGQL statement without executing the statement, while `PROFILE` executes the statement, then output the execution plan as well as the execution profile. You can optimize the queries for better performance with the execution plan and profile.
+
+## Execution Plan
+
+The execution plan is determined by the execution planner in the Nebula Graph query engine.
+
+Execution planner processes the parsed nGQL statements into actions. An action is the smallest unit that can be executed. A typical action could be fetching all neighbors of a given vertex, getting the properties of an edge, or filtering vertices or edges based on the given condition. Each action is assigned to a operator which performs the action.
+
+For example, a simple `SHOW TAGS` statement is processed as two actions and assigned to a `Start` operator and a `ShowTags` operator, while a more complex `GO` statement may be processed as more than 10 actions and assigned to operators of the same number.
+
+## Syntax
+
+* `EXPLAIN`
+
+```nGQL
+EXPLAIN [format="row" | "dot"] <your_nGQL_statement>
+```
+
+* `PROFILE`
+
+```nGQL
+PROFILE [format="row" | "dot"] <your_nGQL_statement>
+```
+
+## Output formats
+
+The return message of an `EXPLAIN` or a `PROFILE` statement has two formats, the default format "row" and format "dot". You can use the `format` option to modify the output format.
```

Comment:
the default format "row" and format "dot" ==> the default "row" format and the "dot" format?